src/index.js

```diff
@@ -32,6 +40,7 @@ const ReduxDevTools = require('./plugins/ReduxDevTools')
 
 module.exports = {
   Core,
+  lib,
```

Comment:
not sure how else to expose this. Happy to get suggestions here. I thought about adding it to core, but I didn't go that route because of the future idea of breaking some parts of uppy to its own npm package. If it were in core then it would be against that future idea.

In any case, I think it is important to expose these modules to allow users write their own custom plugins with these helpers. If there's an understanding I am missing, please let me know.

Reply:
I think `server` and `views` keys could be good, so you'd do `Uppy.server.Provider` or `Uppy.views.ProviderView` with the unpkg bundle. With CommonJS you would just require `uppy/lib/server/Provider` so that's not a problem.

Reply:
yeah I thought about that too, but the issues is that plugins are also added on the same level. So how do we distinguish plugins?

like: `Uppy.GoogleDrive` is a plugin, but is `Uppy.server` a plugin? what if we now have a plugin with that name? So I decided to add it to just `lib` and that's the only exception to be aware of.
Otherwise should we move, `Uppy.GoogleDrive` to `Uppy.plugins.GoogleDrive`?

Reply:
:thinking: I guess the difference would be that plugins are ClassCase and `.server`/`.views` are lowercase. We also have `Uppy.Core` which is not a plugin.